sdk/scheduler/src/test/java/com/mesosphere/sdk/http/PodResourceTest.java

```diff
@@ -407,11 +407,11 @@ public void testRestartPodManyRunning() {
         assertEquals("test-0-c", json.getJSONArray("tasks").get(2));
         assertEquals("test-0-d", json.getJSONArray("tasks").get(3));
 
-        verify(mockTaskKiller).killTask(POD_0_TASK_A.getTaskId());
-        verify(mockTaskKiller).killTask(POD_0_TASK_B.getTaskId());
-        verify(mockTaskKiller).killTask(POD_0_TASK_C.getTaskId());
-        verify(mockTaskKiller).killTask(POD_0_TASK_D.getTaskId());
-        verifyNoMoreInteractions(mockTaskKiller);
+        verify(driver).killTask(POD_0_TASK_A.getTaskId());
```

Comment:
So, we are calling the `killTask` on the driver directly instead of using a `TaskKiller` now?

Reply:
No, we're verifying that ultimately the right thing happened.  Ultimately we should really kill the task.